x-pack/plugins/endpoint/public/applications/endpoint/view/hosts/details.tsx

```diff
@@ -170,3 +181,15 @@ export const HostDetailsFlyout = () => {
     </EuiFlyout>
   );
 };
+
+const useHostLogsUrl = (hostId: string): { url: string; appId: string; appPath: string } => {
```

Comment:
Created separate hook case we would like to later use it in other areas of the UI. Would be easier to extract out.